### PR TITLE
Document Native Text Indices

### PR DESCRIPTION
**Pull request by atris**

### CODE REVIEW

File: features/Native Text Indices [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+Historically, most text indices depend on the in memory text index being written to first and then sealed,
+before searches are possible. This limits the freshness of the search, being near real time at best.
+
+Native text indices come with a custom in memory text index, which allows for real time indexing and search.
+
+# Searching Native Text Indices
+A new function, TEXT_CONTAINS, is introduced for supporting text search on native text indices.
+
+```sql
+SELECT COUNT(*) FROM Foo WHERE TEXT_CONTAINS (<column_name>, <search_expression>)
```
Comment (mayankshriv) [inline features/Native Text Indices:38]: Would be good to elaborate using a few examples of `search_expression`.

File: features/Native Text Indices [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+A new feature that native text indices support are real time text search. For REALTIME tables, native text
+indices allow data to be indexed in memory in the text index, while concurrently supporting text searches
+on the same index.
+
+Historically, most text indices depend on the in memory text index being written to first and then sealed,
+before searches are possible. This limits the freshness of the search, being near real time at best.
+
+Native text indices come with a custom in memory text index, which allows for real time indexing and search.
+
+# Searching Native Text Indices
```
Comment (mayankshriv) [inline features/Native Text Indices:34]: Would be good to call out any syntactic/semantic differences from existing text indices, and what's supported what's not.

**Review by mayankshriv — APPROVED** [review]

File: features/Native Text Indices (hunk 2 above)
Comment (atris) [reply features/Native Text Indices:34]: Highlighted the difference in TEXT_CONTAINS details.

**Comment by atris** [comment]
@mayankshriv Thanks for reviewing. If this looks ok, could you please merge the same?

**Review by mayankshriv — APPROVED** [review]

**Comment by kkrugler** [comment]
Hi @atris & @mayankshriv - I think it's very important to highlight that text analysis is done using the StandardAnalyzer, and is currently not configurable.

**Comment by atris** [comment]
Good point. Mind adding that? While you are at it, it would be worthwhile to add it in Lucene based text index section as well.

**Pull request merged**